src/composables/use-theme.ts

```
@@ -0,0 +1,18 @@
+const defaultThemeColor = '#caae88';
+
+export const useTheme = () => {
+  const themeCookie = useCookie<'dark' | 'light'>('poupe-color-mode', {
+    maxAge: 1000 * 60 * 60 * 24,
+    sameSite: 'lax',
+    default: () => 'light',
+  });
+
+  const darkMode = computed<boolean>({
+    get: () => themeCookie.value === 'dark',
+    set: (value: boolean) => themeCookie.value = value ? 'dark' : 'light',
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)